packages/components/skeleton/tests/skeleton.test.tsx

```diff
@@ -39,4 +38,15 @@ describe("<Skeleton />", () => {
     )
     expect(screen.getByTestId("Skeleton")).toHaveClass("ui-skeleton--loaded")
   })
+
+  test("contents are rendered correctly", () => {
+    render(
+      <Skeleton>
+        <span>hoge</span>
+        <p>huga</p>
+      </Skeleton>,
+    )
+    expect(screen.getByText("hoge")).toBeInTheDocument()
+    expect(screen.getByText("huga")).toBeInTheDocument()
+  })
```

Comment:
Is this test necessary?

Reply:
I don't think this is a width and height test because the test passes even if the width and height are at the wrong values. It would be difficult to write a test for width and height, so I think it could be removed.

Reply:
I thought you were absolutely right. I removed this test.